crates/wit-parser/src/ast.rs

```diff
@@ -47,6 +47,7 @@ impl<'a> Ast<'a> {
                             WorldItem::Type(_) => {}
                             WorldItem::Import(Import { kind, .. }) => imports.push(kind),
                             WorldItem::Export(Export { kind, .. }) => exports.push(kind),
+                            WorldItem::Include(i) => f(None, &i.from, Some(&i.names))?,
```

Comment:
I don't think that `Some(&i.names)` is correct to pass along here because that's intended for "names used from this interface" whereas this list of names is instead "renamed imports/exports"

Reply:
Got it, does it mean I should just pass a `None`?